Adyen/Core/Components/AbstractPersonalInformationComponent/AbstractPersonalInformationComponent.swift

```diff
@@ -19,6 +19,9 @@ open class AbstractPersonalInformationComponent: PaymentComponent, PresentableCo
 
     /// :nodoc:
     public let apiContext: APIContext
+
+    /// :nodoc:
+    public let adyenContext: AdyenContext
```

Comment:
maybe we should add `var adyenContext: AdyenContext { get }` to the `PaymentComponent` and `ActionComponent` protocols? so that if we miss it in one component we get a compile time error.

Reply:
Yes! Good point. I'm gonna add this on following current PR in which I'm working on!